### ISSUE TITLE
[StructuralMechApp] Minor error when printing PParameters.json

### ISSUE THREAD

**Issue by AlejandroCornejo** [KratosMultiphysics/GiDInterface#927]
The latest version of the ptype is writing:

~~~json
{
            "python_module" : "assign_vector_variable_process",
            "kratos_module" : "KratosMultiphysics",
            "process_name"  : "AssignVectorVariableProcess",
            "Parameters"    : {
                "model_part_name" : "Structure.DISPLACEMENT_fixed_z",
                "variable_name"   : "DISPLACEMENT",
                "interval"        : [0.0,"End"],
                "constrained"     : [true,false,false],
                "value"           : [0.0,[null],[null]]
            }
        }
~~~

as is giving problems since the `null `should not be a list and Kratos crashes reading

**Comment by rubenzorrilla** [comment]
I think @Marco1410 had the same problem not long ago.

**Comment by loumalouomega** [comment]
> The latest version of the ptype is writing:
> 
> ```json
> {
>             "python_module" : "assign_vector_variable_process",
>             "kratos_module" : "KratosMultiphysics",
>             "process_name"  : "AssignVectorVariableProcess",
>             "Parameters"    : {
>                 "model_part_name" : "Structure.DISPLACEMENT_fixed_z",
>                 "variable_name"   : "DISPLACEMENT",
>                 "interval"        : [0.0,"End"],
>                 "constrained"     : [true,false,false],
>                 "value"           : [0.0,[null],[null]]
>             }
>         }
> ```
> 
> as is giving problems since the `null `should not be a list and Kratos crashes reading

This is after validation?, how can I reproduce this?

**Comment by Marco1410** [comment]
Yes @rubenzorrilla, I had the same problem in the interface for compressible fluid problems. When I create a new problem this value appears. This does not happen with the examples.


![212921078-f06bf949-c693-441e-9269-be1f6c81e7b8](https://user-images.githubusercontent.com/49286443/227177475-e9f7f834-8bf4-48fc-a843-af2cded40b29.jpg)

**Comment by rubenzorrilla** [comment]
@jginternational I think this is way urgent...

**Comment by jginternational** [comment]
I thought this was already fixed. grrrrr

**Comment by jginternational** [comment]
Fixed both in master branch and in release 9.3.2
@AlejandroCornejo if any student has downloaded the version, please make them delete it and download it again via GiD internet retrieve
